### ISSUE TITLE
Frame's shadow doesn't translate with the Frame by TranslatTo method (iOS)

### ISSUE THREAD

**Issue by mshwf** [xamarin/Xamarin.Forms#9990]
### Description
When translating `Frame` with `TranslateTo` method, its shadow doesn't translate with it.
### Steps to Reproduce

1. Set `HasShadow` of a `Frame` to `true`
2. Move the `Frame` away from its position using `TranslateTo` method.

```
<StackLayout>
    <Button Text="Move" Clicked="BtnMove" HorizontalOptions="Center"/>
    <StackLayout WidthRequest="500">
        <Frame HorizontalOptions="Center" 
               CornerRadius="30"
               BackgroundColor="Red" HasShadow="True" x:Name="frm"
               WidthRequest="15"
               HeightRequest="15"/>
    </StackLayout>
</StackLayout>
```

```
int x = 60;
private void BtnMove(object sender, EventArgs e)
{
    frm.TranslateTo(x, 0);
    x = -x;
}
```

### Expected Behavior
The whole frame is translated to the new position (the Frame and its shadow)
### Actual Behavior
The Frame moves to the new position, but the shadow is still in the same original position.

### Basic Information

- Version with issue: multiple versions (didn't notice a version without the issue), but this tested against last XF version 4.5.0.396
- Last known good version:
- IDE: VS for Windows (16.5.0)
### Screenshots

![1](https://user-images.githubusercontent.com/10940379/76857447-1c588800-685e-11ea-8aaf-b0e47f4ba274.PNG)
![2](https://user-images.githubusercontent.com/10940379/76857451-1d89b500-685e-11ea-8f17-a1e26002d568.PNG)

## Reproduction link

https://drive.google.com/file/d/133UWKhmZRHSSqej__JTg7jq8ljHhmZvr/view?usp=sharing

### Workaround
Set `HasShadow` to `false`

**Comment by jfversluis** [comment]
Probably has to do with #7518 and the additional layer not picking up on the translate.

**Comment by mshwf** [comment]
I've added reproduction link to the issue

**Comment by jfversluis** [comment]
Adding the reproduction to the issue here so we won't lose it. Confirmed this is happening with latest source as well. iOS only.

[FrameShadowIssue.zip](https://github.com/xamarin/Xamarin.Forms/files/4343963/FrameShadowIssue.zip)

**Comment by maexsp** [comment]
We experienced the same issue. Not yet fixed in actual version 4.5.0.396
